doc/changes/changes_0.3.0.md

```diff
@@ -0,0 +1,12 @@
+# dynamodb-virtual-schema 0.3.0, released 2020-07-08
+ 
+## Summary
+This pre-release introduces selection & projection pushdown for DynamoDB query generation 
+and query parallelization. 
+
+## Features / Enhancements
+ 
+* Selection push-down to for DynamoDB requests
```

Comment:
Do you have issue numbers?

Reply:
yes. I added them now.

In itelij Idea it looks however not so nice:
![grafik](https://user-images.githubusercontent.com/18247545/86887583-116d8000-c0f9-11ea-88e6-56241d8857d9.png)

Reply:
No, github displays it in a normal way. It's a usual way to reference a ticket